frontend/app/framework/utils/error.ts

```diff
@@ -15,6 +18,7 @@ export class ErrorDto {
         public readonly inner?: any
     ) {
         this.displayMessage = formatMessage(message, details);
+        this.localizer = LocalizerService.getInstance();
```

Comment:
I think we should not use singleton here. I have only added it to the backend code because it was not possible otherwise. But I though we should just use the localizer in the components, when the error is displayed, not in errrodto